docs/concepts/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Knative concepts"
+linkTitle: "concepts"
+weight: 100
+type: "docs"
+---
+
+{{% readfile file="README.md" %}}
```

Comment:
We actually don't need to do this anymore - you can add the content straight into the _index file and delete the README.

FWIW, if the only thing in this file is duck typing, I'd just make the file be called ducktyping.md instead of index, since I think this is a big enough doc to have it's own section like Concepts (folder) > Duck typing

Reply:
ACK, as a followup (maybe on your side?) :D

Reply:
lol I suppoooooose so.... 😜